buddyboss/buddyboss-platform
# PROD-753 Fix - Copy Paste error in replies and comments of the activity feed

### All Submissions:


Fixes #2856 

### How to test the changes in this Pull Request:

1. Go to the 'demo site' activity feed page
2. Copy an URL or text
3. Click on the Comment section and paste the URL, it will be fine
4. Now paste the link or text again in the Comments or reply section
5. It should not duplicate now

### Proof Screenshots or Video
https://www.loom.com/share/1c089b0774ab48dc861169a37c86f9bb

### Other information:
